README.md

```diff
@@ -17,6 +17,14 @@
 
 This is the FAST monorepo, containing web component packages, tools, examples, and documentation. FAST tech can be used à la carte or as a suite to build enterprise-grade websites, applications, components, design systems, and more.
 
+**tl;dr;**
+
+* Create reusable UI components based on web component standards.
+* Use our standard library of pre-built web components in your apps.
+* Choose between two *adaptive* design systems: Fluent Design and FAST Frame.
```

Comment:
There may soon only be FAST design system as msft may move to Fluent UI

Reply:
I'm thinking that I could turn these into links and link out to the fluent repo for that since it's still a design system built by our team.